FeedStoreChallenge/ManagedObjects/ManagedCache+CoreDataProperties.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ManagedCache+CoreDataProperties.swift
+//  FeedStoreChallenge
+//
+//  Created by Vita Solomina on 2021-05-01.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+extension ManagedCache {
+	@nonobjc public class func fetchRequest() -> NSFetchRequest<ManagedCache> {
+		return NSFetchRequest<ManagedCache>(entityName: "ManagedCache")
+	}
+
+	@NSManaged public var timestamp: Date
+	@NSManaged public var images: NSOrderedSet
```

Comment:
These properties are a `CoreDataFeedStore` implementation detail so they should not be public

Please make it `internal`.
```suggestion
	@NSManaged var timestamp: Date
	@NSManaged var images: NSOrderedSet
```